course-lessons/python-fundamentals/python-fundamentals-variables.md

```diff
@@ -0,0 +1,200 @@
+
+A key characteristic of variables is that once a variable is created within your code (e.g. **Jupyter Notebook** file), the value of that variable will remain the same and can be reused throughout the code.
+
+However, you can overwrite an existing variable if you create a new variable with the same name. 
+
+While there are occassions in which you might want to overwrite an existing variable to assign a new value to it, you want to make sure that you give variables both clear and distinct names to avoid *accidentally* overwriting variable values.  
+
+
+## Numeric Variables
+
+In **Python**, you can create variables to store numeric values such as  integers (`int`) (i.e. whole numbers) and floating point numbers (`float`) (i.e. decimal values).  
+
+For more advanced math applications, you can also use variables to work with <a href="https://docs.python.org/3/library/stdtypes.html#numeric-types-int-float-complex" target="_blank">complex numbers (see Python documentation for more details)</a>. 
+
+As described previously, you do not need to define which numeric type you want to use to create a variable. 
+
+For example, you can create a `int` variable called `boulder_precip_in`, which contains the rounded up value for the average annual precipitation in inches (in) in Boulder, Colorado. 
+<!-- #endregion -->
+
+```python
+boulder_precip_in = 21
+
+boulder_precip_in
+```
+
+You could also create a `float` variable to capture the data using decimal units. 
+
+```python
+boulder_precip_in = 20.68
+
+boulder_precip_in
+```
+
+Notice that in these examples, `boulder_precip_in` was overwritten by the second variable definition because both definitions assigned the same name to the variable. 
+
+
+## Text String Variables
+
+To create a variable containing a text string (`str`), you need to use quotations (`""`) around the value to identify the value as a text string (e.g. `variable_name = "text"`). 
+
+While in **Python** the single quote (`''`) and the double quote (`""`) are used interchangeably (see the official <a href="https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str" target="_blank">Python Docs</a> for more examples), it is good to choose one option and use it consistently. This textbook uses the double quote (`""`) for identifying text strings (`str`). 
+
+Using quotes, you can create `str` variables that contain a single word or many words, including punctuation.
+
+```python
+city = "Boulder"
+
+city
+```
+
+```python
+city = "Boulder, CO"
+
+city
+```
+
+```python
+city = "Boulder, CO is the home of the University of Colorado, Boulder campus."
+
+city
+```
+
+### Interactive Activity
+
+Assign values to variables! In the cell below, create a variable called `int_var` that has an integer as its value. Then create another variable called `float_var` that has a float as its value, and another variable called `string_var` that has a string as its value. 
```

Comment:
I love this activity!  Let's make these examples some real data though, so that they can create some variables for another city (since the examples on this page create variables for Boulder) or additional variables for Boulder, such as average low or high temperatures. 

Ex: [average annual precipitation for New York city](https://www.usclimatedata.com/climate/new-york/new-york/united-states/usny0996): 46.23 inch

Ex: [temperature variables for Boulder](https://psl.noaa.gov/boulder/)

Reply:
I made a minor change here, so that the string variable created is text: "NYC".